### ISSUE TITLE
scenes directory

### ISSUE THREAD

**Issue by vapidbabble** [pxl-th/Trace.jl#32]
I do not see the "scenes" directory referred to in the Readme in the github project. Is this generated?

**Comment by lazarusA** [comment]
yes, indeed. Now is in `/docs/code/`.  But, we should fine some time merge this branch https://github.com/pxl-th/Trace.jl/tree/sd/tmp, with has a lot improvements and GPU support (Metal was still failing, last time I checked).

But, yes for the purpose of this issue, `scenes` does not longer exists. And `/code` will be gone, once the documentation is finish. Now we only have the shadows example there: https://pxl-th.github.io/Trace.jl/dev/shadows
